src/main/java/frc/robot/subsystems/gyro/Gyro.java

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+package frc.robot.subsystems.gyro;
+
+import org.littletonrobotics.junction.Logger;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Gyro extends SubsystemBase
+{
+    private final GyroIO                _io;
+    public final GyroIOInputsAutoLogged _inputs = new GyroIOInputsAutoLogged();
```

Comment:
This guy should be private:

```suggestion
    private final GyroIOInputsAutoLogged _inputs = new GyroIOInputsAutoLogged();
```